localstack/services/ses/ses_starter.py

```diff
@@ -128,6 +131,32 @@ def get_identity_verification_attributes(self):
 
     email_responses.get_identity_verification_attributes = get_identity_verification_attributes
 
+    backend_send_email_orig = SESBackend.send_email
+
+    def send_email_save_contents(self, source, subject, body, destinations, region):
+        message = backend_send_email_orig(self, source, subject, body, destinations, region)
+
+        data_dir = config.DATA_DIR
+        if data_dir == '':
```

Comment:
Would this also work if `config.DATA_DIR` has some other falsey value, e.g., `None`?

I'd suggest we change this to:
```
data_dir = config.DATA_DIR or config.TMP_FOLDER
```

Reply:
Uh... good catch! :) My one would now probably crash for a weird reason. When I think about it:
Should we also check if the path in DATA_DIR actually exists?
Ah.. no, it's already dealt with in config, so if it is a non-false value I think it is save to assume it is there :)

I'll change that in an update.